### DISCUSSION TITLE
Allow searching by ID

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9403] [Ideas]
> Created by: Timothy Kelty (timkelty@gmail.com) on 2015/04/02 12:33:05 +0000
> Votes at time of UserVoice import: 3

-----------------------------------

There is currently no way to search by ID.
Using "field searching", e.g. id::"123" doesn't work either.

http://craftcms.stackexchange.com/questions/17433/search-by-id-in-the-cp

**Comment by angrybrad** [comment]
> Posted by Nick (nick@jtnetinc.com) on 2017/01/18 04:51:14 +0000
-----------------------------------

Would love to see this sorted, given the current option to use ID as a column on the entries listing pages. Many of our clients use IDs for much of their internal organization and entry workflow discussions.

**Comment by terryupton** [comment]
Is this planned for anytime soon? 
Or is there any kind of work around available now?

**Comment by brandonkelly** [comment]
Yes, actually, it’s in the works :)

**Comment by espensgr** [comment]
Any progress on this? i'm having this request from some of our costumers as well

**Comment by brandonkelly** [comment]
Took a detour but we should be able to get this added for 3.8.
